src/libsync/theme.cpp

```diff
@@ -533,7 +549,22 @@ QColor Theme::wizardHeaderBackgroundColor() const
 QPixmap Theme::wizardHeaderLogo() const
 {
 #ifdef APPLICATION_WIZARD_USE_CUSTOM_LOGO
-   return QPixmap(hidpiFileName(":/client/theme/colored/wizard_logo.png"));
+#ifdef APPLICATION_ICON_SET
+    const auto useSvg = QByteArray(APPLICATION_ICON_SET).toUpper() == QByteArrayLiteral("SVG");
+#else
+    const auto useSvg = false;
+#endif
+
+    if (useSvg) {
+        const auto maxHeight = 64;
+        const auto maxWidth = 2 * maxHeight;
+        const auto icon = QIcon(":/client/theme/colored/wizard_logo.svg");
```

Comment:
@er-vin IMHO it makes sense to put this **":/client/theme/colored/wizard_logo** into a separate const, and then concat **svg/png** to it.
This is mostly for readability than to avoid duplication. It is easy to spot the difference when there is just **png or svg** converted to a short-named variable, instead of a long path to an image name with its extension in the end.

Reply:
done